extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/HttpSecurityRecorder.java

```diff
@@ -224,6 +224,14 @@ public void created(BeanContainer container) {
     public Supplier<FormAuthenticationMechanism> setupFormAuth() {
 
         return new Supplier<FormAuthenticationMechanism>() {
+
+            private final Function<String, String> START_WITH_SLASH = new Function<String, String>() {
```

Comment:
Why is this a function object and not just a static method?

Reply:
I meant to avoid lambda & method reference and chose wrong solution. Fixed, thank you.